internal/core/metadata/main.go

```diff
@@ -25,14 +25,15 @@ import (
 	pkgHandlers "github.com/edgexfoundry/edgex-go/internal/pkg/bootstrap/handlers"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/telemetry"
 
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+	"github.com/gorilla/mux"
+
```

Comment:
Goland IDE inserts them in this way and collapses the import section so these breaks are not readily apparent. I'll fix the one in the PR, but IMO this is not something we should care about.

Reply:
@farshidtz , I pushed commit to sort imports.

Reply:
Thanks Lenny. You are right and this doesn't change anything in the final binaries. But then we could have the same argument for code formatting.

Apart from readability, this keeps the history cleaner. You can see that the diff now doesn't include lines that you didn't really wanna change. This also reduced merge conflicts when someone changes other parts of the code and makes other imports.

For Goland, you may able to prevent that from happening: https://stackoverflow.com/a/70668482/3041544